src/doublyLinkedList.js

```diff
@@ -0,0 +1,103 @@
+class Node{
+  constructor(data, next = null,prev = null) {
+    this.data = data
+    this.next = next
+    this.prev = prev || null
+  }
+  data() { return this.data }
+  next() { return this.next }
+  previous() { return this.prev }
+}
+
+export default class DoublyLinkedList{
+  constructor() {
+    this.head = null
+    this._length = 0
+  }
```

Comment:
It is good to use as small amount of data as possible. I do believe , however, that a doubly linked list is prime real-estate for setting up a reference to your tail, to make your insert and pop algorithms a little bit faster.